tools/src/h5dump/h5dump.c

```diff
@@ -326,11 +326,15 @@ usage(const char *prog)
                    "      (Alternate compact form of subsetting is described in the Reference Manual)\n");
     PRINTVALSTREAM(rawoutstream, "\n");
     PRINTVALSTREAM(rawoutstream, "--------------- Option Argument Conventions ---------------\n");
-    PRINTVALSTREAM(rawoutstream, "  D - is the file driver to use in opening the file. Acceptable values\n");
     PRINTVALSTREAM(
         rawoutstream,
-        "      are \"sec2\", \"family\", \"split\", \"multi\", \"direct\", and \"stream\". Without\n");
-    PRINTVALSTREAM(rawoutstream, "      the file driver flag, the file will be opened with each driver in\n");
+        "  D - is the file driver to use in opening the file. Acceptable values are available from\n");
+    PRINTVALSTREAM(
+        rawoutstream,
+        "      "
+        "https://portal.hdfgroup.org/documentation/hdf5-docs/registered_virtual_file_drivers_vfds.html.\n");
```

Comment:
@bmribler @loricooperhdf  Are the portal.hdfgroup.org urls in this PR going to remain, or will they need to be replaced?

Reply:
They include "Hosted on GitHub Pages, so I assume they're probably going to be around.

Reply:
> @bmribler @loricooperhdf Are the portal.hdfgroup.org urls in this PR going to remain, or will they need to be replaced?

This url, https://portal.hdfgroup.org/documentation/hdf5-docs/registered_virtual_file_drivers_vfds.html, is new and I assume it'll stay.  Lori can confirm.